patches/minecraft/net/minecraft/block/BambooBlock.java.patch

```diff
@@ -1,16 +1,16 @@
 --- a/net/minecraft/block/BambooBlock.java
 +++ b/net/minecraft/block/BambooBlock.java
-@@ -100,10 +100,11 @@
+@@ -100,10 +100,12 @@
  
     public void func_225542_b_(BlockState p_225542_1_, ServerWorld p_225542_2_, BlockPos p_225542_3_, Random p_225542_4_) {
        if (p_225542_1_.func_177229_b(field_220266_f) == 0) {
 -         if (p_225542_4_.nextInt(3) == 0 && p_225542_2_.func_175623_d(p_225542_3_.func_177984_a()) && p_225542_2_.func_226659_b_(p_225542_3_.func_177984_a(), 0) >= 9) {
 +         if (p_225542_2_.func_175623_d(p_225542_3_.func_177984_a()) && p_225542_2_.func_226659_b_(p_225542_3_.func_177984_a(), 0) >= 9) {
              int i = this.func_220260_b(p_225542_2_, p_225542_3_) + 1;
--            if (i < 16) {
-+            if (i < 16 && net.minecraftforge.common.ForgeHooks.onCropsGrowPre(p_225542_2_, p_225542_3_, p_225542_1_, p_225542_4_.nextInt(3) == 0)) {
+             if (i < 16) {
++               if (net.minecraftforge.common.ForgeHooks.onCropsGrowPre(p_225542_2_, p_225542_3_, p_225542_1_, p_225542_4_.nextInt(3) == 0)) {
                 this.func_220258_a(p_225542_1_, p_225542_2_, p_225542_3_, p_225542_4_, i);
-+               net.minecraftforge.common.ForgeHooks.onCropsGrowPost(p_225542_2_, p_225542_3_, p_225542_1_);
++               net.minecraftforge.common.ForgeHooks.onCropsGrowPost(p_225542_2_, p_225542_3_, p_225542_1_);}
```

Comment:
Changes like this really don't make much sense at least with the fact of putting a bracket on the same line for closing makes it very hard to read/see what is going on given things like:
```java
if (x) doY(); else {
```
are not things that get done to reduce patch size as it makes it hard to see that it is an alternative branch of a thing. So then putting the bracket on its own line would be a larger patch than this already is (or at least about the same size), except is harder to read for someone looking at the code. With how you have it currently with the bracket being on the same ending line it makes it nearly impossible to read what is going on easily.

Reply:
> ```java
> if (x) doY(); else {
> ```
None of the edits I made follow that format.

The reason I chose to make a bracket and then not indent in cases like in BambooBlock is because I saw it in the patches made in `Entity#changeDimension`.
In this case I don't think it sacrifices much readability, as this block is very short and there are no alternative code paths to get confused with.

Reply:
I am not talking about indenting I am talking about the closing bracket. Which is hard to read, and why I gave an example of a similar thing that we don't do because of readability, even if it reduces the patch size.